### ISSUE TITLE
wrong Git tags

### ISSUE THREAD

**Issue by hboutemy** [airlift/airbase#460]
airbase is marked as successful Reproducible Builds: https://github.com/jvm-repo-rebuild/reproducible-central/blob/master/content/io/airlift/airbase/README.md

but if you look at buildspecs, you'll see that on each release, I need to manually get the Git commit that was used to do the release to Maven Central, because it's never the commit associated to the Git tag

it would be nice if in future releases, the Git tag was correct, please :)

**Comment by wendigo** [comment]
This seems to be an issue with our release automation. I’ll investigate it next week

**Comment by hboutemy** [comment]
@wendigo just a reminder that Git tags are still wrong, I need to adjust manually the commit id on each release

**Comment by wendigo** [comment]
@hboutemy yes, I'm aware and I have it on my ever-growing to-do list. I'll try to tackle it next week as I need to add similar automation to other projects (trinodb/s3-proxy and gateway to name a few)

**Comment by wendigo** [comment]
nah, that doesn't work either. I don't get it why the JReleaser is creating improver tag since the HEAD points to the `prepare release` commit

**Comment by wendigo** [comment]
This is https://github.com/jreleaser/jreleaser/issues/1692

**Comment by hboutemy** [comment]
ping? can I help?

**Comment by wendigo** [comment]
@hboutemy I need to change the release flow as the jreleaser is unable to handle this one properly. I have it on my long todo list :(

**Comment by wendigo** [comment]
@hboutemy new release workflow based on njord (instead of jreleaser) created 273 with a correct tag
